examples/customresources/demos/complexmodule/moduletest/module_test.go

```diff
@@ -115,18 +115,18 @@ func TestComplexModule(t *testing.T) {
 
 	// Base is a custom component, but built-in API. It also depends on built-in motors, so tests dependencies.
 	t.Run("Test Base", func(t *testing.T) {
-		res, err := rc.ResourceByName(base.Named("base1"))
-		test.That(t, err, test.ShouldBeNil)
-		mybase := res.(base.Base)
-
-		res, err = rc.ResourceByName(motor.Named("motor1"))
+		res, err := rc.ResourceByName(motor.Named("motor1"))
```

Comment:
Any reason we're switching order around here?

Reply:
From Zack's PR:
![image](https://github.com/viamrobotics/rdk/assets/71458/81d9e205-8c6e-4510-9837-d93a8fafe160)